lib/ui.rb

```diff
@@ -1,25 +1,27 @@
+require 'clear_screen_printer'
+
 class Ui
   attr_reader :input, :output
-  CLEAR_SCREEN = "\e[H\e[2J"
 
   def initialize(input, output)
     @input = input
     @output = output
+    @printer = ClearScreenPrinter.new(output)
   end
 
-  def menu(options)
-    output.puts "#{CLEAR_SCREEN} :::Contacts management::: \n\nPlease choose a menu option:\n\n"
+  def display_menu(options)
+    printer.print(" :::Contacts management::: \n\nPlease choose a menu option:\n\n")
     options.each do |number, option|
-      output.puts "#{number} - #{option}"
+      output.puts "#{number} - #{format_for_display(option)}"
     end
     output.print"\n---> "
```

Comment:
Shouldn't all of the printing be done through the printer? Then wouldn't need the `@output` as a parameter...